### ISSUE TITLE
NTR, pulmo: Detection of anti-GM-CSF antibodies

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5568]
**Preferred term label:**
Detection of anti-GM-CSF antibodies in broncho-alveolar lavage

**Synonyms**
Detection of auto-antibodies against granulocyte-macrophage colony stimulating factor (GM-CSF)

**Definition (free text, please give PubMed ID)**
The presence of anti-GM-CSF auto-antibodies seems to define an autoimmune disease underlying most of the adult sporadic type of pulmonary alveolar proteinosis (PAP). 
Comment: this finding may be central to the pathogenesis of adult sporadic pulmonary alveolar proteinosis (PAP). It is usually not found in pediatric forms of PAP. 

(doi: 10.1136/thx.2004.021329 )

**Parent term (use hpo.jax.org/app)**
Abnormal biochemical composition of broncho-alveolar fluid

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 610910, OMIM: 300770, OMIM: 614370 

**Your nano-attribution (ORCID)**
crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
Anti-GM-CSF autoantibody positivity (https://hpo.jax.org/app/browse/term/HP:0020050)--in serum
Decision -- no need for the BAL term
Todo -- annotate (check) the three diseases above for annotations. 

e.g. https://www.ncbi.nlm.nih.gov/pubmed/17287477 for OMIM: 610910

Update the definition.

**Comment by pnrobinson** [comment]
moving issue to annotation tracker
